tests/s3/copy_object/test_copy_object_tag.py

```diff
@@ -551,22 +550,22 @@ def test_44814(self):
         self.copy_obj_di_check(self.bucket_name1, self.object_name1, self.bucket_name2,
                                self.object_name2, put_etag=put_etag, copy_etag=copy_etag)
         self.log.info("Step 8: Verification of etag and metadata for data integrity check")
-        self.log.info("ENDED: copy-object with destination tag set same as source object"
-                      "(--tagging-directive=REPLACE)")
+        self.log.info("ENDED: Test simple copy-object with destination tag set same as source"
+                      " object (--tagging-directive=REPLACE)")
 
     @pytest.mark.parallel
     @pytest.mark.s3_ops
     @pytest.mark.s3_object_copy
-    @pytest.mark.tags("TEST-44815")
-    def test_44815(self):
+    @pytest.mark.tags("TEST-44817")
+    def test_44817(self):
         """
-        TEST-44815 :Test simple copy-object with source object having max number of tags(N<=10)
+        TEST-44817 :Test simple copy-object with source object having max number of tags(N<=10)
```

Comment:
instead of max number of tags, you can say tags <=10 here. because we already have test with same name above for n=10.

Reply:
done